test/checks/keyboard/banner-is-top-level.js

```diff
@@ -0,0 +1,89 @@
+describe('banner-is-top-level', function () {
+	'use strict';
+
+	var fixture = document.getElementById('fixture');
+
+	var checkSetup = axe.testUtils.checkSetup;
+	var shadowSupported = axe.testUtils.shadowSupport.v1;
+
+	afterEach(function () {
+		fixture.innerHTML = '';
+	});
+
+	it('should return false if banner landmark is in main element', function() {
+		var main = document.createElement('main');
+		var banner = document.createElement('div');
+		banner.setAttribute('role','banner');
+		main.appendChild(banner);
+		fixture.appendChild(main);
+		assert.isFalse(checks['banner-is-top-level'].evaluate(banner));
+	});
+
+	it('should return false if banner landmark is in main element', function () {
+		var main = document.createElement('main');
+		var banner = document.createElement('div');
+		banner.setAttribute('role','banner');
+		main.appendChild(banner);
+		fixture.appendChild(main);
+		assert.isFalse(checks['banner-is-top-level'].evaluate(banner));
+	});
+
+	it('should return false if banner landmark is in div with role main', function () {
+		var main = document.createElement('div');
+		main.setAttribute('role','main');
+		var banner = document.createElement('div');
+		banner.setAttribute('role','banner');
+		main.appendChild(banner);
+		fixture.appendChild(main);
+		assert.isFalse(checks['banner-is-top-level'].evaluate(banner));
+	});
+
+	it('should return false if header is not sectioning element and in div with role search', function () {
```

Comment:
This test name is confusing. How about something like "it should return false if header is inside of div with role search"? It doesn't seem important that the header isn't a sectioning element, more that the header isn't *inside* of a sectioning element so it's not a banner anymore (and it should be top level). But we don't have to go into that level of detail in a test name.

Reply:
Updated the test name